### ISSUE TITLE
Immersive Engineering Tesla Crashing

### ISSUE THREAD

**Issue by Lost-Ninja** [Darkosto/InvasionIssues#312]
I have a mob trap killing Blazes/PigZombies/Skeles, while mostly I do the killing myself, when I'm unable to do so, I have a tesla from immersive engineering do it. However occasionally this happens:

https://pastebin.com/2PKxNkV2

Think this is the 2nd such crash in about 15 hours of play. So not exactly common. 

I'll crosspost this to the IE tracker but I think the IE dev has finished up work on 1.10.2

**Comment by Darkosto** [comment]
Yeah, I believe there's an update since last update I made for the pack ( I think). I'll see what I can do before this next pack update. Thank you :)
